### ISSUE TITLE
ansi-tests: bit on bitarrays not being simple fails

### ISSUE THREAD

**Issue by kpoeck** [drmeister/clasp#426]
````
(bit (make-array 5 :element-type 'bit :fill-pointer 2 :initial-element 1) 1)

Debugger received error of type: SIMPLE-TYPE-ERROR
The value of CORE::BIT-ARRAY is #*11, which is not of type (SIMPLE-ARRAY BIT).
````

**Comment by kpoeck** [comment]
Fixed by pr #427

**Comment by kpoeck** [comment]
Fixed in dev with the merged PR
